packages/theming/atlas/scripts/build.js

```diff
@@ -0,0 +1,34 @@
+const { exec } = require("shelljs");
+
+function build() {
```

Comment:
You can inline this function, it is needed only when async

Reply:
✅